CarpetX/src/prolongate_3d_rf2.hxx

```diff
@@ -462,20 +462,20 @@ extern prolongate_3d_rf2<CC, CC, CC, MINMOD, MINMOD, MINMOD, 1, 1, 1, FB_NONE>
 
 // Hermite interpolation
 
-extern prolongate_3d_rf2<VC, VC, VC, HERMITE, HERMITE, HERMITE, 1, 1, 1,
+extern prolongate_3d_rf2<VC, VC, VC, POLY, POLY, POLY, 1, 1, 1,
                          FB_NONE>
```

Comment:
if these are actually the same as Langrangian, does it make sense to completely remove them instead and error out?